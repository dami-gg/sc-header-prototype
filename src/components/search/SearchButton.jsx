import React, {Component} from 'react';

import './Search.scss';

class SearchButton extends Component {
  render() {
    return (
        <button className="dc-btn dc-search-form__btn">
          <i className="dc-icon dc-icon--search dc-icon--interactive"></i>
        </button>
    );
  }
}

export default SearchButton;
