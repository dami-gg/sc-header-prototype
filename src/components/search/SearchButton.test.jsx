import React from 'react';
import renderer from 'react-test-renderer';
import SearchButton from './SearchButton';

describe('SearchButton component', () => {
  it('should render correctly', () => {
    let tree = renderer.create(<SearchButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});