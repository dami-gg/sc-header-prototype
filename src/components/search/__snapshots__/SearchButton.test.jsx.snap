exports[`SearchButton component should render correctly 1`] = `
<button
  className="dc-btn dc-search-form__btn">
  <i
    className="dc-icon dc-icon--search dc-icon--interactive" />
</button>
`;
